src/main/resources/application.conf

```diff
@@ -383,6 +383,10 @@ mantis {
     # if true, account storage will use Ethereum-specific format for storing keys/value in MPT (32 byte)
     # if false, generic storage for arbitrary length integers will be used
     eth-compatible-storage = true
+
+    # true: gas limit is constant for all blocks in the blockchain and equal to genesis block gas limit
+    # false: the default Ethereum formula will be applied (based on genesis block)
+    constant-block-gas-limit = false
```

Comment:
After some thought, I think it will be better if we specify the number here, rather than basing on genesis block gas limit (`null` for std Ethereum formula). That way we can adjust the limit without discarding the whole blockchain.